lib/query/compiler.js

```diff
@@ -19,7 +19,7 @@ const {
   reduce,
   has,
 } = require('lodash');
-const uuid = require('uuid');
+const uuid = require('uuid').v4;
```

Comment:
```suggestion
const uuid = require('uuid/v4');
```

Reply:
Actually, this is not going to work and was actively discouraged by `uuid`, as it couples import to specific directory structure. What I should have used instead is alias for { v4 } require, but I was too lazy to look up syntax for that; will do it now, since that would definitely look cleaner.

Reply:
Ok, I see